<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>639</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>969</width>
    <height>639</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>969</width>
    <height>639</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>色彩达人</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../imandqss.qrc">
    <normaloff>:/game/assets/color/logo.png</normaloff>:/game/assets/color/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>971</width>
     <height>681</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/game/assets/color/1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="color" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>331</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#template_matching{
border-top-left-radius:20px;
border-bottom-left-radius:20px;
}
QLabel#Template{
border-top-right-radius:20px;
border-bottom-right-radius:20px;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="template_matching">
      <property name="minimumSize">
       <size>
        <width>166</width>
        <height>331</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>166</width>
        <height>331</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Template">
      <property name="minimumSize">
       <size>
        <width>166</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="scores">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>110</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>291</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="spinBox_R">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QSpinBox
{
	border:1px solid #242424;
}
 
QSpinBox::up-button,QSpinBox::down-button
{
	width:0px;
}</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_R">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider#horizontalSlider_R::groove:horizontal{
height:12px;
left:0px;
right:0px;
border:0px;
border-radius:6px; 
background:#808080;
}
QSlider#horizontalSlider_R::handle:horizontal{
width:24px;
margin:-6px 0px;
background:#66d2ff;
border-radius:12px;
}
QSlider#horizontalSlider_R::sub-page:horizontal{
border:0px; 
border-radius:6px;
background:qlineargradient(spread:pad, x1:0,y1:1, x2:1,y2:1, 
stop:0 rgba(231,80,229,255),
stop:1 rgba(7,208,255,255))
}</string>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="spinBox_G">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QSpinBox
{
	border:1px solid #242424;
}
 
QSpinBox::up-button,QSpinBox::down-button
{
	width:0px;
}</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_G">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider#horizontalSlider_G::groove:horizontal{
height:12px;
left:0px;
right:0px;
border:0px;
border-radius:6px; 
background:#808080;
}
QSlider#horizontalSlider_G::handle:horizontal{
width:24px;
margin:-6px 0px;
background:#66d2ff;
border-radius:12px;
}
QSlider#horizontalSlider_G::sub-page:horizontal{
border:0px; 
border-radius:6px;
background:qlineargradient(spread:pad, x1:0,y1:1, x2:1,y2:1, 
stop:0 rgba(231,80,229,255),
stop:1 rgba(7,208,255,255))
}</string>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QSpinBox" name="spinBox_B">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QSpinBox
{
	border:1px solid #242424;
}
 
QSpinBox::up-button,QSpinBox::down-button
{
	width:0px;
}</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_B">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider#horizontalSlider_B::groove:horizontal{
height:12px;
left:0px;
right:0px;
border:0px;
border-radius:6px; 
background:#808080;
}
QSlider#horizontalSlider_B::handle:horizontal{
width:24px;
margin:-6px 0px;
background:#66d2ff;
border-radius:12px;
}
QSlider#horizontalSlider_B::sub-page:horizontal{
border:0px; 
border-radius:6px;
background:qlineargradient(spread:pad, x1:0,y1:1, x2:1,y2:1, 
stop:0 rgba(231,80,229,255),
stop:1 rgba(7,208,255,255))
}</string>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>470</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>3</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="results">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="login" native="true">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>0</y>
     <width>1161</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="login_background">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>-70</y>
      <width>971</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/game/assets/color/login1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_8" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="start_game">
       <property name="styleSheet">
        <string notr="true">position: absolute;
width: 309px;
height: 62px;
top: 0;
left: 0;
background: rgb(38, 120, 255);
border-radius: 4px;
position: absolute;
width: 180px;
height: 30px;
top: 16px;
left: 64px;
color: rgba(255, 255, 255, 0.9);
font-family: Noto Sans SC;
font-size: 30px;
font-weight: 400;
line-height: 24px;
letter-spacing: -1px;
text-align: center;
position: absolute;
width: 309px;
height: 62px;
top: 523px;
left: 73px;
background: rgb(38, 120, 255);
border: 4px solid rgba(255, 255, 255, 0.6);
border-radius: 12px;</string>
       </property>
       <property name="text">
        <string>开始游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exit_game">
       <property name="styleSheet">
        <string notr="true">position: absolute;
width: 309px;
height: 62px;
top: 0;
left: 0;
background: rgb(38, 120, 255);
border-radius: 4px;
position: absolute;
width: 180px;
height: 30px;
top: 16px;
left: 64px;
color: rgba(255, 255, 255, 0.9);
font-family: Noto Sans SC;
font-size: 30px;
font-weight: 400;
line-height: 24px;
letter-spacing: -1px;
text-align: center;
position: absolute;
width: 309px;
height: 62px;
top: 523px;
left: 73px;
background: rgb(38, 120, 255);
border: 4px solid rgba(255, 255, 255, 0.6);
border-radius: 12px;</string>
       </property>
       <property name="text">
        <string>退出游戏</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_9" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="simple">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>简单</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="difficulty">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>困难</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Game_Description">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(117, 171, 255);</string>
    </property>
    <property name="text">
     <string>游戏 说明：
      规定时间内猜出匹配的RGB值</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="novice">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>新手教程</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="count_down">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QWidget" name="prepare_box" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>641</height>
    </rect>
   </property>
   <widget class="QPushButton" name="prepare">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>510</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/game/assets/color/2.png);</string>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="color_" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>331</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#template_matching_2{
border-top-left-radius:20px;
border-bottom-left-radius:20px;
}
QLabel#Template_2{
border-top-right-radius:20px;
border-bottom-right-radius:20px;
}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="template_matching_2">
       <property name="minimumSize">
        <size>
         <width>166</width>
         <height>331</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>166</width>
         <height>331</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 131, 131);
font: 20pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>模板</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Template_2">
       <property name="minimumSize">
        <size>
         <width>166</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>需匹配的
模板</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Microsoft YaHei UI&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>时间</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Microsoft YaHei UI&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>分数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>widget_7</zorder>
  <zorder>background</zorder>
  <zorder>color</zorder>
  <zorder>scores</zorder>
  <zorder>widget_5</zorder>
  <zorder>confirm</zorder>
  <zorder>results</zorder>
  <zorder>count_down</zorder>
  <zorder>prepare_box</zorder>
  <zorder>login</zorder>
 </widget>
 <resources>
  <include location="../../imandqss.qrc"/>
 </resources>
 <connections/>
</ui>
